### PR TITLE
Update wording in testing-practice lesson

### PR DESCRIPTION
**Pull request by xari4808**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
- [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1.Describe the changes made and include why they are necessary or important:

Reason

- Assignment description asks to write functions yet there is a task to write an object
- Inconsistent wording between tasks
- Singular form of the word "instruction" is used when linking to Jest Docs which has more than one instructions

#### 2. Related Issue

keyword #XXXXX

### CODE REVIEW

File: javascript/testing/testing-practice.md [hunk 1]
```diff
@@ -4,25 +4,26 @@ Let's practice!  This testing thing really is not that difficult, but it _is_ qu
...
-Write tests for the following functions, and then make the tests pass!
+Write tests for the following, and then make the tests pass!
 
-1. `capitalize(string)` takes a string and returns that string with the first character capitalized.
+1. A `capitalize` function which takes a string and returns it with the first character capitalized.
 
-2. `reverseString(string)` takes a string and returns it reversed.
+2. A `reverseString` function which takes a string and returns it reversed.
 
 3. A `calculator` object that contains the basic operations: `add`, `subtract`, `divide`, and `multiply`.
```
Comment (thatblindgeye) [inline javascript/testing/testing-practice.md:13]: This line should be updated to match verbiage of the other tasks:

```suggestion
3. A `calculator` object that contains functions for the basic operations: `add`, `subtract`, `divide`, and `multiply`. Each of these functions should take two numbers and return the correct calculation.
```

File: javascript/testing/testing-practice.md [hunk 2]
```diff
@@ -4,25 +4,26 @@ Let's practice!  This testing thing really is not that difficult, but it _is_ qu
...
 
-4. Caesar Cipher. [Read about it on this website](http://practicalcryptography.com/ciphers/caesar-cipher/)
+4. A `caesarCipher` function. [Read about it on this website](http://practicalcryptography.com/ciphers/caesar-cipher/).
```
Comment (thatblindgeye) [inline javascript/testing/testing-practice.md:15]: Same as above, some verbiage addition:

```suggestion
4. A `caesarCipher` function that takes a string and returns it with each character "shifted". Read more about how a Caesar cipher works on [this website](http://practicalcryptography.com/ciphers/caesar-cipher/).
```

**Review by thatblindgeye — COMMENTED** [review]
Thanks for making these updates. I added some requested changes after this comment, but I also have two others here:

1) For the capitalize and reverse tasks, change "which takes" to "that takes".

2) Add four spaces at the start of lines 44-46 and lines 50-54 (the two code snippets at the end of the lesson, including the lines with the 3 backticks). This will fix the issue of two separate lists being created when it should be a single list with two numbered items "1." and "2.". If you need to check if you did it write, copy+paste the contents of the lesson into this link to preview it: https://www.theodinproject.com/lessons/preview

**Comment by xari4808** [comment]
@thatblindgeye Thank you so much for the feedback. I've committed the requested changes.

**Comment by thatblindgeye** [comment]
Awesome, thank you for making these updates!

**Pull request merged**